cli-plugins/apps/create-workload.hbs.md

```diff
@@ -57,26 +57,15 @@ Tanzu Application Platform supports creating a workload from an existing local p
     - `--source-image` is the registry path where the local source code will be uploaded as an image.
 
     **Exclude Files**
-    When working with local source code, you can exclude files from the source code to be uploaded within the image by creating a file `.tanzuignore` at the root of the source code. You can find the options available to specify the workload in the command reference for [`workload create`](command-reference/tanzu-apps-workload-create.md), or run `tanzu apps workload create --help`.
+    When working with local source code, you can exclude files from the source code to be uploaded within the image by creating a file `.tanzuignore` at the root of the source code.
 
     The file must contain a list of file paths to exclude from the image including the file itself and the directories must not end with the system path separator (`/` or `\`).
 
-    If the file contains files or directories that are not in the source code, they are ignored.
-
-    If a line in the file starts with a `#` hashtag , the line is ignored.
-
-    **Example**
-
-    ```
-    # This is a comment
-    this/is/a/folder/to/exclude
-
-    this-is-a-file.ext
-    ```
+    More info regarding .tanzuignore file can be found in the [how-to guide](how-to-guides.hbs.md#tanzuignore-file).
```

Comment:
More info regarding .tanzuignore file can be found in the [tanzuignore-file section of the how-to guide](how-to-guides.hbs.md#tanzuignore-file).